paymentsheet/src/test/java/com/stripe/android/paymentsheet/ui/FakeAddPaymentMethodInteractor.kt

```diff
@@ -0,0 +1,17 @@
+package com.stripe.android.paymentsheet.ui
+
+import kotlinx.coroutines.flow.StateFlow
+import org.mockito.Mockito
+
+internal object FakeAddPaymentMethodInteractor : AddPaymentMethodInteractor {
+    override val state: StateFlow<AddPaymentMethodInteractor.State>
+        get() = Mockito.mock()
```

Comment:
Can we use a real mutable state flow + dummy data? Or even throw?

Reply:
yep! I just updated this to throw -- if we need to use it in tests in the future, we can add functionality to it